tutorials/phasorpy_components.py

```diff
@@ -100,5 +105,103 @@
 plt.tight_layout()
 plt.show()
 
+# %%
+# Graphical solution for the contribution of multiple components
+# --------------------------------------------------------------
+#
+# The graphical method for resolving the contribution of two or
+# three components (pairwise) to a phasor coordinate is based on
+# the quantification of moving circular cursors along the line
+# between the components, demonstrated in the following animation:
```

Comment:
Totally agree, the code for the animation is too long and kind of distracting. Do you know a way to hide the code but still make visible the execution? since the code for generating the animation is not so relevant since its just for visualization purposes. I searched for a way to do this but was unable to find a convincing approach with sphinx.

Reply:
I have not tried any of these: you could try [hiding lines of code](https://sphinx-gallery.github.io/stable/configuration.html#hiding-code-blocks), or move the code block to an external module and import it, or just include an image of the animation. The nicest would be to [collapse](https://sphinx-toolbox.readthedocs.io/en/stable/extensions/collapse.html) the entire code block but that requires a third-party extension and I'm not sure it works within the sphinx gallery.

Reply:
Hiding the lines of code worked quite well. It is not perfect since it still has a blank cell code with nothing inside that doesn't seem to disappear. Also it says that is not recommended as it is not a very good practice, but it does the job for now. Let me know what you think about it when you can.

Reply:
I just noticed that the animation code is missing in the generated Jupyter notebook. I also tried to get the other methods (module import and collapse extension) working in the tutorial without success. Maybe it's better to leave the code block visible for now?

Reply:
Ok, yes maybe it's better to leave the code block visible until we find a better solution. Yes, with the sphinx flags the code is hidden altogether but the empty code is visible, which is not ideal.